packages/app-democracy/src/useCounter.ts

```diff
@@ -7,8 +7,8 @@ import { useApi, trackStream } from '@polkadot/react-hooks';
 
 export default function useCounter (): number {
   const { api, isApiReady } = useApi();
-  const proposals = trackStream<any[]>(isApiReady ? api.derive.democracy.proposals : undefined, []);
-  const referenda = trackStream<any[]>(isApiReady ? api.derive.democracy.referendums : undefined, []);
+  const proposals = trackStream<any[]>(isApiReady ? api.derive.democracy && api.derive.democracy.proposals : undefined, []);
```

Comment:
This is always there. (But could fail)

Reply:
Yes I just checked and you're correct. I'll push a commit reverting the change